libr/core/cfile.c

```diff
@@ -314,21 +313,20 @@ static int r_core_file_do_load_for_debug(RCore *r, ut64 baseaddr, const char *fi
 	}
 #endif
 	int fd = cf ? cf->fd : -1;
+	RBinOptions bo = {
+		.offset = 0LL,
+		.baseaddr = baseaddr,
+		.rawstr = false,
+		.xtr_idx = xtr_idx,
+		.iofd = fd,
+		.loadaddr = UT64_MAX,
+	};
 
-	bo = r_bin_options_new (0LL, baseaddr, false);
```

Comment:
should we kill the options_new then? its usually handy for bindings to have methods to construct structs rather than having to build them by hand., from C its fine like this imho. but then we should probably specify that the binoptions is const in the rbinopen signature

Reply:
i removed the options_new api, yes. and yes, i will add `const`.